src/card/card.js

```diff
@@ -23,7 +23,11 @@ export default (data, cardStyling, selectionObj) => {
   const topColor = colorUtils.getDarkColor(backgroundColor);
   const fontColor = getFontColor(cardStyling, backgroundColor);
   const attributes = data.attributes || {};
-  let html = `<div class="sn-org-card-title">${attributes.label || data.id}</div>`;
+  let html = '<div class="sn-org-card-content">';
+  if (attributes.imageURL) {
+    html += `<img class="sn-org-card-image" src="${attributes.imageURL}"/>`;
```

Comment:
think the URL needs to be validated so there is no possibility for script injection.

Reply:
Well that would be a big check I think. Also everyone can see which checks we do, but more important which checks we don't do

Reply:
hmm, I was thinking a encode URL would do it, but googling doesn't give me any of that. 
Think we can look at how it is done in the T&I object, `web/assets/client/services/media-tool/formatters/link.js`